src/client/extension.ts

```diff
@@ -11,11 +11,12 @@ import { fileURLToPath, sleep } from './utils'
 import { getDefaultPowerShellPath, getPlatformDetails } from './platform';
 import settings = require("./settings");
 import * as process from './process';
-import { EvaluateRequestMessage, IEvaluateRequestArguments } from "./messages";
+import { EvaluateRequestMessage, IEvaluateRequestArguments, GetHelpRequestMessage } from "./messages";
 
 async function getCurrentSelection(mode: string) {
     let doc = await workspace.document
 
+    // This doesn't actually get the selection... It gets the lines... As though the mode was V-LINE and not VISUAL...
```

Comment:
@TylerLeonhardt or @yatli Was it intentional to get the lines instead of the selection? I was hoping to use getCurrentSelection for the Get-Help wrapper in that we could get the selected command, but we can't if we get the whole line.

Reply:
would you like to get the word under the cursor?

Reply:
Well yes, but also in my mind getCurrentSelection would be to get the currently selected text when in VISUAL mode. 

Given the line:

```
Write-Host this is a test
```

a selection of `Write-Host` would ideally return just `Write-Host`. As it is currently, it returns the entire line.

Reply:
Yeah unfortunately, @yatli had to hack the getCurrentSelection method to work (it works pretty well!). coc.nvim supplies a “get current selection” equivalent but I couldn’t get it to work and requires some work to get an idea for what’s going wrong:
https://github.com/neoclide/coc.nvim/issues/933

Reply:
maybe, we should distinguish `mode` "v" from "V"

Reply:
From what I can tell there are effectively 3 Visual modes: V-LINE ("V"), V-BLOCK ("Ctrl+v"), and VISUAL ("v"). Should we distinguish between V-Block and normal Visual? I would think in V-Block mode, we'd want to get from start to end and possibly ignore that it's a block (for evaluation I can't think of a scenario where we would want to evaluate just the block).

Reply:
Had a second thought and realized this:

Executing a part of something would be dangerous.
I wouldn't want someone to execute:
```
["rm -rf /"]foo/bar
```

Reply:
Ah yes, so for evaluation then line based makes sense. Perhaps I'll either create a get-help specific one, or add a mode specific to Get-Help. As using Get-Help you'll not want the entire line, but just the selected command.